pennylane_lightning_gpu/lightning_gpu.py

```diff
@@ -15,7 +15,8 @@
 This module contains the :class:`~.LightningGPU` class, a PennyLane simulator device that
 interfaces with the NVIDIA cuQuantum cuStateVec simulator library for GPU-enabled calculations.
 """
-from typing import List
+import itertools
```

Comment:
```suggestion
```
Please double-check, but I don't think we use the `intertools` library here.

Reply:
Ah yes, you are correct, that functionality was moved into the `_serialize.py` instead. I'll remove it